### PR TITLE
Implemented file/species based train_validation split 

### PR DESCRIPTION
**Pull request by Sean1572**

### CODE REVIEW

**Review by benjamin-cates — APPROVED** [review]
Looks good to me
Sad pylint failure tho :(

File: classification/dataset.py [hunk 1]
```diff
@@ -96,8 +96,15 @@ def verify_audio_files(self) -> bool:
...
-            if spectrogram.isnan().any():
+            try:
+                spectrogram, _ = self[i]
+                if spectrogram.isnan().any():
+                    bad_files.append(i)
+            except Exception as e:
+                #Sam if this broke something I'm sorry
+                #If there is any error, mostly likely due to headers confusing the decoder
+                #Ignore this file
+                print_verbose(e, verbose=self.CONFIG.verbose)
```
Comment (sprestrelski) [inline classification/dataset.py]: this breaks, as self doesn't have a CONFIG reference

File: classification/train.py [hunk 2]
```diff
@@ -218,7 +220,7 @@ def init_wandb(CONFIG: Dict[str, Any]):
     Initialize the weights and biases logging
     """
     run = wandb.init(
-        project="birdclef-2023",
+        project="acoustic-species-reu2023",
```
Comment (sprestrelski) [inline classification/train.py:158]: should make this a parameter in default parser probably

**Review by sprestrelski — DISMISSED** [review]

File: classification/train.py (hunk 2 above)
Comment (benjamin-cates) [reply classification/train.py:158]: yeah probably

File: classification/dataset.py (hunk 1 above)
Comment (benjamin-cates) [reply classification/dataset.py]: Sean be fixing this

File: classification/train.py (hunk 2 above)
Comment (Sean1572) [reply classification/train.py:158]: later

**Pull request closed without merging**